app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+class Product < ApplicationRecord
+    validates :product_name, presence: true
```

Comment:
refactor this to `validates_presence_of :product_name, :price, :description`

Reply:
Changed to
```
Rails.application.routes.draw do
  get 'product/index'
  root controller: :products, action: :index
  resources :products, only: [:index]
end
```

Reply:
You dont need `get 'product/index'`

Reply:
fixed